src/main/scala/se/nimsa/dcm4che/streams/DicomParsing.scala

```diff
@@ -68,6 +115,10 @@ trait DicomParsing {
   }
 
   def isFileMetaInformation(tag: Int) = (tag & 0xFFFF0000) == 0x00020000
+  def isMediaStorageSOPClassUID(tag: Int) = tag == 0x00020002
```

Comment:
Most tags are defined in `org.dcm4che3.data.Tag`. Replacing this method and the two below with code like `if (tag == Tag.MediaStorageSOPClassUID)` may be just as clear.